### DISCUSSION TITLE
Error when trying to send a Tweet

### DISCUSSION THREAD

**Discussion by Snej-Corleone** [jef/streetmerchant#2887] [Help] [answered]
Hello, 

I have an issue when trying to automatically send a tweet when the Bot found a product. I'm getting this error:

```
 "0": {
    "message": "You currently have Essential access which includes access to Twitter API v2 endpoints only. If you need access to this endpoint, you’ll need to apply for Elevated access via the Developer Portal. You can learn more here: https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api#v2-access-leve",
    "code": 453
  }
}
```

Can anybody help me? I heard that it has something to do with the Twitter API.

**Comment by Mr-Tech-13** [comment]
Im not exactly sure because I don't have twitter, but this may help
https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api#v2-access-leve

**Comment by gjj07** [comment]
I am having the exact same problem

**Comment by Mr-Tech-13** [comment, marked as answer]
You need to enable elevated access in the developer settings
